python/tests/test_fast_inference_sdpa.py

```diff
@@ -0,0 +1,114 @@
+# Created by Brian Keene, 2024
```

Comment:
@bpkeene we are putting acknowledgements in the [ACKNOWLEDGEMENTS.md](https://github.com/ml-explore/mlx/blob/main/ACKNOWLEDGMENTS.md) file rather than have author / maintainer lists in individual source code files.

Reply:
sgtm! will move to ACKNOWLEDGEMENTS.md & add Argmax, Inc. :)

Reply:
Awesome! Please take liberty to add anyone who contributed to this on your team (e.g. Atila as well)